test/jdk/java/awt/print/PrinterJob/ValidatePage/ValidatePage.java

```diff
@@ -63,8 +63,7 @@ public class ValidatePage extends Frame implements Printable {
     Label myImageableHeightLabel;
     Label myOrientationLabel;
 
-    private static String GetOrientation(int o)
-    {
+    private static String GetOrientation(int o) {
```

Comment:
```suggestion
    private static String getOrientation(int o) {
```
The method name should start with *a lower-case letter*. And the call sites need updating too.